ManualText.txt

```diff
@@ -829,61 +838,23 @@ If cameras go down
 •	If the camera for the Cashier's office goes down, immediately notify the officer and follow the same steps to try and fix it. This is because a lot of money is handled through this office and it is important for security reasons to have the cameras up. 
 Confidentiality
 •	As with all other things in the office, what is seen on the cameras is to be kept confidential and only shared with other members of the department on a need to know basis. 
 •	At no time should you be using your phone (or other device) to take pictures or video of what is on a camera feed (including if you see a close friend on one of the cameras). 
 •	Capturing images of people on the camera system is an invasion of privacy and is a serious offense. 
 
-6.4	Scanner
+7.4	Scanner
 Speakers for our radio scanner are on the window sill behind the dispatch computer. We monitor several local channels but the most important are the Wenham PD and FD channels. You are permitted to turn down the volume on scanner during high traffic times (meaning multiple stations are talking as is GCPD) if it is distracting you from listening to the Gordon radio. However it is NEVER all right to turn the volume all the way down. The scanner should also be audible over any movies, games, or music you are listening to while in the office. Monitoring the scanner may seem pointless, but Wenham may try to contact us over the radio or you may hear about an incident near the campus (i.e. Grapevine Rd., Parsons Hill, or Iron Rail) that you or the officer should be aware of. It is also possible in a world of cell phones that there is an emergency called in to 911 by a cell phone on campus that the officer will need to respond to.
 Any time you overhear anything on the scanner that the officer should be aware of, radio him/her to make sure they heard it as well.  This includes BOLOs (Be On the Look Out).
 
-6.5	RRC Back Doors and Mirror
+7.5	RRC Back Doors and Mirror
 At night and on windy days the back door to the office door is sometimes locked by the officer.  People will buzz the door for access.  You can let them in by pressing and holding the “Rodger Door” button above the gate buttons on the dispatch console. Occasionally, people will think the door is locked when it is not and will buzz in accordingly. Motion them to come in the door.   If it is late at night and the door has not been locked and you would feel more comfortable if it were, radio the officer and (s)he will lock it for you.
 
 The convex mirror in the office will allow you to see the customer service counter from the back of the office. However, it is most useful when you are sitting in front of the computer; you can view the desk, hallway, and back door without having to turn around. Check the hallway when you hear the back door open. If an unauthorized party goes upstairs, notify the officer immediately.
 
-6.6	Security Alarms
```

Comment:
We still need s section on security alarms, temperature alarms, and power outages.